green_cli/tx.py

```diff
@@ -217,8 +219,8 @@ def _filter_utxos(utxo_filter, utxos):
         if  address and address == utxo.get('address', None):
             selected.append(utxo)
             continue
-        if txhash == '*' or txhash == utxo['txhash']:
-            if not pt_idx or pt_idx == '*' or int(pt_idx) == utxo['pt_idx']:
+        if txhash == '*' or txhash == 'all' or txhash == utxo['txhash']:
+            if not pt_idx or pt_idx == 'all' or pt_idx == '*' or int(pt_idx) == utxo['pt_idx']:
```

Comment:
nit: comparing in the same order would read better, or perhaps a utility function `_is_all` could be used.